lib/build.js

```diff
@@ -266,10 +285,29 @@ export default {
 
       let stdout = '';
       let stderr = '';
+      let liveErroMatchQueue = '';
+      let liveErroMatchHandle = target.liveErrorMatch ? function (d) {
+        d = liveErroMatchQueue + d;
+        liveErroMatchQueue = '';
+        let index = d.lastIndexOf('\n');
+        if (index >= 0 && index === d.length - 1) {
+          that.errorMatcher.parseAdd(d);
+        } else {
+          liveErroMatchQueue = d.substring(index + 1);
+          that.errorMatcher.parseAdd(d.substring(0, index));
+        }
```

Comment:
Now you're adding lines, but there are many reasons why this doesn't work:

  * An error match may span many lines
  * Newlines are different on different OSes and also different tools within OSes (e.g. running cygwin on Windows may have a tool outputting `\n` as new lines even though `\r\n` is the line separator on windows). Build can't make any assumptions on the line endings.